language/java/language.go

```diff
@@ -129,7 +139,12 @@ func (l *Language) Mistakes(logger *log.Logger, repositoryPath string) (mistakes
 	})
 	if err != nil {
 		if output != "" {
-			return extractMistakes(output), nil
+			mistakes := extractMistakes(output)
+			for i, mistake := range mistakes {
+				mistakes[i] = strings.TrimPrefix(mistake, repositoryPath)
+			}
+
+			return mistakes, nil
```

Comment:
Add comment.

I guess we will have the same problem with windows again where the reported maven compile errors are with unix slashes. Not sure if worth to look into.